src/components/sdk/MeetingControls/docs/AudioInputVFControl.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import AudioInputVFControl from '../AudioInputVFControl';
+
+<Meta title="SDK Components/MeetingControls/AudioInputVFControl" />
+
+# AudioInputVFControl
+
+The `AudioInputVFControl` component renders a `ControlBarButton` with pop over menu options to select through multiple audio input options,
+and provides the option of voice focus on the bottom. If you want to apply the voice focus feature, try this `AudioInputVFControl`. 
+If not, `AudioInputControl` is a better choice. This component should be used within VoiceFocusProvider and meetingProvider. 
```

Comment:
- Suggest to move "If not, `AudioInputControl` line to bottom most after you explain everything about `AudioInputVFControl` first.
- Backtick VoiceFocusProvider.
- meetingProvider -> `MeetingProvider`.

Reply:
Updated